pkg/api/common.go

```diff
@@ -133,10 +138,31 @@ func ParseFlags(fs *flag.FlagSet, cfg *Config) *Config {
 }
 
 func Validate(cfg *Config) error {
```

Comment:
Considering what is happening in this function, it is a bit misleading name. IMHO validation shouldn't mutate or create new objects.

Reply:
done